transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/ObservationListSpec.groovy

```diff
@@ -1,37 +0,0 @@
-package tests.rest.v2
```

Comment:
This is a test for the observationsList endpoint, which I removed altogether. I though it had already been removed a long time ago, but it wasn't. All users should use the /observations endpoint for this functionality.